flake.nix

```diff
@@ -416,6 +417,8 @@
                 haskellPackages.fourmolu
                 nodePackages.prettier
                 nodePackages.eslint
+                unzip
+                (pkgs.lib.optionalString withChromium chromium)
```

Comment:
`packages` is a list of Nix packages, so you can't use `optionalString`. You should use
```
packages = with pkgs; [
  # ...
] ++ pkgs.lib.lists.optional
  withChromium
  chromium;
```

But this should actually go into `nix/default.nix` (see my other comment)

Reply:
ok. 

> so you can't use optionalString

Seems to work though.

Reply:
@ngua Thanks for the input. Can you please have a look again?